pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -184,10 +184,11 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		}
 		pluginsMap[name] = p
 
-		// a weight of zero is not permitted, plugins can be disabled explicitly
+		// a weight of zero (or unset) is not permitted, plugins can be disabled explicitly
 		// when configured.
-		f.pluginNameToWeightMap[name] = int(pg[name].Weight)
-		if f.pluginNameToWeightMap[name] == 0 {
+		if int(pg[name].Weight) != 0 {
```

Comment:
this also looks like a no-op diff now

Reply:
can revert this diff, it is the same as lines 189-192 above

Reply:
sorry, missed that